action.yml

```diff
@@ -81,7 +81,7 @@ outputs:
     description: "The commit hash if the SDK was regenerated"
 runs:
   using: "docker"
-  image: "Dockerfile"
+  image: "docker://speakeasysagar/sdk-generation-action:v8"
```

Comment:
I'm not 100% sure this is how to define this here, did you find documentation on it? I'm just looking now to validate but from memory it should just be `speakeasysagar/sdk-generation-action`

Reply:
Here is the metadata syntax documentation: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsuses

```
runs:
  using: "composite"
  steps:
...
...
    # References a docker public registry action
    - uses: docker://gcr.io/cloud-builders/gradle
    # Reference a docker image published on docker hub
    - uses: docker://alpine:3.8
```

Reply:
ah cool 👍